src/main/java/org/veupathdb/service/eda/ss/model/reducer/FilteredValueFile.java

```diff
@@ -27,6 +27,17 @@
   private T next;
   private boolean hasStarted;
 
+  /**
+   * Constructs an instance of a FilteredValueFile.
```

Comment:
I wonder if FilteredValueIterator would be a better name.   The current name suggests we are creating a file, while actually we just happen to be reading from a file.

Reply:
Sure, updated the name